server/api/contributors.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * API endpoint for fetching contributors.
+ * @example
+ * // Example usage:
+ * // GET /api/contributors
+ */
+// eslint-disable-next-line
+export default defineEventHandler(async (event) => {
+  // we're going to use our key
+  const { githubAppAPIKey } = useRuntimeConfig(event).githubAPI
+
+  // and the api endpoint
+  const CONTRIBUTORS_ENDPOINT = 'https://api.github.com/repos/devedmonton/DES-Website/collaborators'
+
+  // make the request
+  try {
+    // await the collaborators endpoint.
+    const response = await fetch(CONTRIBUTORS_ENDPOINT, {
```

Comment:
we can use [`$fetch`](https://nitro.unjs.io/guide/fetch) that way converting to `json` and some other things are not needed

Reply:
Will do I was unaware of this from nitro TBH. Thanks @arashsheyda